sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/pubsub/PubsubIO.java

```diff
@@ -731,8 +731,9 @@ public static Write<PubsubMessage> writeMessagesDynamic() {
    */
   public static Write<String> writeStrings() {
     return Write.newBuilder(
-            (String string) ->
-                new PubsubMessage(string.getBytes(StandardCharsets.UTF_8), ImmutableMap.of()))
+            (ValueInSingleWindow<String> string) ->
```

Comment:
Perhaps change the parameter name from "string" to something like "stringAndWindow" or something better.

Reply:
done